doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function does not work on AIX versions older than 7.1. It can still
+be called but will return `UV_ENOSYS`.
+
```

Comment:
Bear with us @gibfahn, the bike shed is not quite yet the right shade ;-) ...

```js
*Note*: This function does not work correctly on AIX versions older than 7.1 in that it will
always return the value `UV_ENOSYS`.
```